cmd/client_changefeed.go

```
@@ -482,6 +483,11 @@ func newCreateChangefeedCommand() *cobra.Command {
 			if err != nil {
 				return err
 			}
+			var out bytes.Buffer
+			err = json.Indent(&out, []byte(infoStr), "", "\t")
```

Could you return the error when it is not nil?

---

done.